misc/config_tools/upgrade_scenario_xml

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+function usage() {
+	echo "Usage: $0 <scenario XML> [<version of the given scenario XML>]"
+	echo "Upgrade the given scenario XML to align with the current version of ACRN."
+	echo ""
+	echo "Example: $0 logical_partition.xml v2.3"
+	echo "    The output file is logical_partition-v2.3.xml"
+}
+
+file=$1
+ver=$2
+
+if [[ "${file}" = "" ]]; then
+	usage
+	exit 1
+fi
+
+if [[ ! -f "${file}" ]]; then
+	echo "${file}: No such file"
+	exit 1
+fi
+
+if [[ ! "${ver}" =~ v[0-9]+.[0-9]+ ]]; then
```

Comment:
Should we restrict this to 2.3 and above?
```suggestion
if [[ ! "${ver}" =~ v[2-9]+.[3-9]+ ]]; then
```

Reply:
Nice try.  This pattern change though would not allow v3.1
Have you verified the pattern match works?

Reply:
I'll keep this regex for generic versions and use another check to ensure the given version is at least v2.3.